### ISSUE TITLE
Atualizar React

### ISSUE THREAD

**Issue by isacjunior** [Tautorn/boilerplate-react-redux#2]
Visto que a nova versão do **React** já está disponível, seria interessante atualizar o projeto para tal versão. Pois melhorias como diminuição do tamanho da lib e Fragments e Strings podem trazer melhores benefícios pra quem for utilizar o boilerplate.

**Comment by Tautorn** [comment]
Concordo com você, mas isto ainda não foi feito porque nem todas as libs atuais do projeto possuem suporte para a versão 16 do react.
